.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish distribution to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: Build and publish distribution to PyPi
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Comment:
This is still publishing to PyPI for 3.7 and 3.8?

Reply:
It'll still be compatible with 3.8; I haven't really seen packages on PyPi with multiple tarballs and wheels for a single version